docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,139 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+*coala* has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+Let us understand how to write tests for ``MyBear`` in ``some_dir``.
+``My Bear`` checks if a file has less than or equal to ``max_number_of_lines``
+lines. ``max_number_of_lines`` by default is 10.
+
+.. code::
+
+    from coalib.results.Result import Result
+    from coalib.bears.LocalBear import LocalBear
+
+
+    class MyBear(LocalBear):
```

Comment:
this is actually an amazing idea, i agree with @abhsag24 :) since we give concrete names to our bears, why shouldnt this one have a name too :) ?